### DISCUSSION TITLE
TOOL IS NOT WORKING!

### DISCUSSION THREAD

**Discussion by UKUNIFOREVER** [maruohon/litematica#657] [Q&A]
What to do if the tool does not work, the panel shows, but I cannot change the modes, as well as move the scheme, to say nothing about selection and so on.
M + T DOES NOT HELP, IT JUST DISCONNECTS THE TOOL AND IT DOESN'T WORK FOR ME!

can someone upload the old version of the mod? What happened to him? There was no such shit before, but now it's just broken

**Comment by UKUNIFOREVER** [comment]
i solved the problem

**Reply by DerRockWolf** [reply]
Could you please add your solution as an answer and mark the discussion as answered.

**Comment by luccahiko-png** [comment]
PLEASE PUT THE FUDGING ANSWER
